lib/services.js

```diff
@@ -168,6 +169,14 @@ const services = {
         // will reassign some of the above values with the ones
         // from the intiation of the multipart upload
         // (for instance, storage class)
+        const validateUserMdRes = validateUserMetadata(metaHeaders);
```

Comment:
I ended up combining this function with the existing util to get the metaheaders, so now it returns the metaHeaders if they are valid.